### DISCUSSION TITLE
Location of GDAL files and search path different

### DISCUSSION THREAD

**Discussion by pfblack** [MaxRev-Dev/gdal.netcore#153] [Q&A] [answered]
Hello I'm using the latest lib from Nuget (3.9.1.247) in a Net 8 test project and there seems to be a mismatch between the path where the library is looking for package data and where it is actually located.  PR 102 changed the location of the data from "runtimes/any-x64/native/gdal-data" to "runtimes/any/native/gdal-data" which conflicts with the data search which still is creating paths with any-x64.

**Comment by MaxRev-Dev** [comment, marked as answer]
I somehow missed that part. I'll patch this. Thank you for pointing that out.

**Reply by pfblack** [reply]
Thanks!
